README.md

```diff
@@ -23,7 +23,7 @@ This is a react based (side) project intended to practice help you in typing fas
 
 ## 📌Live Demo:
 
-![Live Demo](https://github.com/kaiwalyakoparkar/flashtype/blob/master/doc_images/FlashType.avi)
+![Live Demo](https://github.com/kaiwalyakoparkar/flashtype/blob/master/doc_images/FlashType.gif)
```

Comment:
Change link to Live Demo link to (./doc_images/FlashType.gif) so that this can work on any fork

Reply:
I fixed the link to gif